### PR TITLE
recipe owner adjustments

### PR DESCRIPTION
**Pull request by miozune**
Properly show current and previous owners
I needed to call `setOwner` because some of the recipe loaders run on `sAfterGTPostload`

### CODE REVIEW

File: src/main/java/com/github/bartimaeusnek/bartworks/system/material/GT_Enhancement/LuVTierEnhancer.java [hunk 1]
```diff
@@ -141,7 +142,7 @@ private static void addDreamcraftItemListItems(Collection LuVMachines){
     private static void rewriteCraftingRecipes(List<IRecipe> bufferedRecipeList, OrePrefixes[] LuVMaterialsGenerated, Predicate recipeFilter){
         for (OrePrefixes prefixes : LuVMaterialsGenerated) {
 
-            Consumer recipeAction = obj -> LuVTierEnhancer.doStacksCointainAndReplace(((GT_Shaped_Recipe) obj).getInput(),
```
Comment (repo-alt) [inline src/main/java/com/github/bartimaeusnek/bartworks/system/material/GT_Enhancement/LuVTierEnhancer.java:144]: is "cointain" an honest typo or some kind of Bart's humor?

Comment (Dream-Master) [reply src/main/java/com/github/bartimaeusnek/bartworks/system/material/GT_Enhancement/LuVTierEnhancer.java:144]: idk but possible

**Comment by repo-alt** [comment]
So, some recipes actually changed in BW did not have "added by BW" sign?

File: src/main/java/com/github/bartimaeusnek/bartworks/system/material/GT_Enhancement/LuVTierEnhancer.java (hunk 1 above)
Comment (miozune) [reply src/main/java/com/github/bartimaeusnek/bartworks/system/material/GT_Enhancement/LuVTierEnhancer.java:144]: lol I didn't notice, blame Bart. I can change it though

**Comment by miozune** [comment]
Yes, directly editing recipe ingredients cannot trigger owner reload. Plus both original and changer are displayed.

**Review by repo-alt — APPROVED** [review]

**Pull request merged**